bridge_test.go

```diff
@@ -51,6 +53,47 @@ func TestHandlePacket(t *testing.T) {
 				&TimerEvent{
 					metricName: "foo",
 					value:      200,
+					labels:     map[string]string{},
+				},
+			},
+		}, {
+			name: "datadog tag extension",
+			in:   "foo:100|c|#tag1:bar,tag2:baz,tag3,tag4",
+			out: Events{
+				&CounterEvent{
+					metricName: "foo",
+					value:      100,
+					labels:     map[string]string{"tag1": "bar", "tag2": "baz", "tag3": ".", "tag4": "."},
+				},
+			},
+		}, {
+			name: "datadog tag extension with # in all keys (as sent by datadog php client)",
+			in:   "foo:100|c|#tag1:bar,#tag2:baz,#tag3,#tag4",
+			out: Events{
+				&CounterEvent{
+					metricName: "foo",
+					value:      100,
+					labels:     map[string]string{"tag1": "bar", "tag2": "baz", "tag3": ".", "tag4": "."},
```

Comment:
Your second point: the implementer decides on the tags. Same problem as having many variable parts in the metric name, like `stats_counts.production.web67.request_time.signup.ab_test_13.cancelled`. Yes, you can shoot yourself in the foot with it. No, you shouldn't.